tests/functional/report/checkRoutes.js

```diff
@@ -0,0 +1,94 @@
+'use strict'; // eslint-disable-line strict
+const http = require('http');
+const https = require('https');
+const fs = require('fs');
+
+const conf = require('../config.json');
+
+const transportStr = conf.transport;
+const transport = transportStr === 'http' ? http : https;
+
+function makeAgent() {
+    if (transportStr === 'https') {
```

Comment:
You're right, I only realized this after I noticed that the end to end tests were only using scality's backends which won't benefit from this and the tests don't need to run there. I'll remove this.